galaxy.yml

```diff
@@ -10,15 +10,17 @@ authors:
 description: null
 license_file: LICENSE
 tags:
-# tags so people can search for collections https://galaxy.ansible.com/search
-# tags are all lower-case, no spaces, no dashes.
+  # tags so people can search for collections https://galaxy.ansible.com/search
+  # tags are all lower-case, no spaces, no dashes.
   - example1
   - example2
+  # You must have one of the tags below to satisfy Ansible Lint:
+  # ['application', 'cloud', 'database', 'eda', 'infrastructure', 'linux', 'monitoring', 'networking', 'security', 'storage', 'tools', 'windows']
```

Comment:
Can we add a link to ansible-lint's documentation instead of hard-coding their list of "required" tags?

Reply:
Sure, I will add a link to https://ansible.readthedocs.io/projects/lint/rules/galaxy/?h=galaxy+yml#galaxy instead 🙂